pkg/cmd/protoc-gen-gogoroach/main.go

```diff
@@ -14,8 +14,35 @@ import (
 	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
 	"github.com/gogo/protobuf/vanity"
 	"github.com/gogo/protobuf/vanity/command"
+	"regexp"
```

Comment:
I think we want these stdlib imports to go above the others.

Reply:
`gofmt` moves the imports back into alphabetical order if I make this change.

Reply:
Our linter will fail (I think what you want is to introduce a newline between the two blocks).

Reply:
Thanks for the tip about the newline -- I just figured that out myself the hard way :)